vue/storybook-v6/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "vue-sample-v6",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -s public",
+    "dsm-storybook:publish": "dsm-storybook publish",
+    "dsm-storybook:preview": "dsm-storybook preview"
+  },
+  "dependencies": {
```

Comment:
It's not present here currently, but we [previously](https://github.com/InVisionApp/dsm-storybook-example-library/blob/17cfb0fd17de5a79f4786c12911ad8460cd749c7/vue/storybook-v5/package.json#L13) had `@emotion/core` and `@emotion/styled` as dependencies in `package.json`.

This caused issues with Storybook's theming as versions were mismatched in `node_modules`. I couldn't figure out _why_ we were using `emotion` so I removed it and it fixed the issue.

See the issue on Storybook's GitHub issues – ["Uncaught TypeError: Cannot read property 'fonts' of undefined"](https://github.com/storybookjs/storybook/issues/12016#issue-678918975)